### PR TITLE
Adds Reviews

### PR DESCRIPTION
**Pull request by ubahl**
Adds reviews to the Reviews page.

Adds a header and white background to Reviews page. Also adds reviews from the data servlet to the Reviews page. Follows teachme ~/step/walkthroughs/week-3-server/step-3-json-walkthrough.md.

Screenshot 2020-06-26 at 5 20 52 PM

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -19,13 +19,35 @@ function reviewsPage()
...
+  /* alternate attempt which didn't work: 
+  *
+  reviewBox = document.getElementById('white-background');
+  for(var i = 0; i < message.length; i++) {
+    newComment = document.createElement('review-text');
+    newComment.innerText = message[i];
+    newBox = document.createElement('review-box');
+    newBox.appendChild(newComment);
+    reviewBox.appendChild(newBox);
+  } */
```
Comment (ubahl) [inline portfolio/src/main/webapp/script.js:47]: Tried to execute this code in a cleaner manner (the commented code) but could not get it to work. Is there a better way to do this?

Comment (mrboxtobox) [reply portfolio/src/main/webapp/script.js:47]: Sure! Could you try this: https://jsfiddle.net/8r7Lpwtm/7/

```
reviewBox = document.getElementById('white-background');
  for(var i = 0; i < message.length; i++) {
    // `createElement` takes in the tag.
    newComment = document.createElement('div');
    // Use `setAttribute` to set the id.
    newComment.setAttribute('id', 'review-text');
    newComment.innerText = message[i];
    newBox = document.createElement('p');
    newBox.setAttribute('id', 'review-box');
    newBox.appendChild(newComment);
    reviewBox.appendChild(newBox);
  }
```

File: portfolio/src/main/java/com/google/sps/servlets/WelcomeServlet.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  
+    @Override
+    public void init() {
+        welcomeMessages = new ArrayList<>();
+        welcomeMessages.add("Feeling bubbly?");
+        welcomeMessages.add("Looking for some good boba? Look no further.");
+        welcomeMessages.add("Looking for something Tas-tea?");
+        welcomeMessages.add("BOBA TIME!");
+        welcomeMessages.add("Oolonging for some tea?");
+        welcomeMessages.add("Have a taro-fic day!");
```
Comment (mrboxtobox) [inline portfolio/src/main/java/com/google/sps/servlets/WelcomeServlet.java:38]: 😄

**Review by mrboxtobox — APPROVED** [review]
Suggested a way to add the reviews. Hope it helps!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (ubahl) [reply portfolio/src/main/webapp/script.js:47]: oh I get it now! thank you

**Pull request merged**